docs/rules/standard.md

```diff
@@ -184,24 +184,44 @@ Consistent removal (default) or adding of trailing comma's on call site.
      * It makes it easy to add and reorder elements – there is no need to add or delete the comma if you manipulate elements.
      * It simplifies code generation, for example, for object initializers. The last element can also have a comma.
 
+!!! note
+    Trailing comma on call site is automatically disabled if [Wrapping](#wrapping) rule (before 0.48.0, [Indentation](#indentation) rule) is disabled. Because it cannot provide proper formatting with unwrapped calls. (see [Dependencies](./dependencies.md))
+
+    ```kotlin
+    processFoo(Foo(
+        a = 3,
+        b = 4,
+    ),) // it's weird to insert "," between unwrapped (continuated) parenthesis
+    ```
+
```

Comment:
Thanks, the previous code block is quite ambiguous for why it exists.
What I want to do is prove that `trailing comma w/o wrapping looks weird" by these code blocks.
So I think the bellow code block is more appropriate for my purpose.
```kotlin
// with wrapping
processFoo(
    Foo(
        a = 3,
        b = 4,
    ),
)
// w/o wrapping
processFoo(Foo(
    a = 3,
    b = 4,
),) // it's weird to insert "," between unwrapped (continued) parenthesis
```

Reply:
I described it more precisely in [0e0ad02](https://github.com/pinterest/ktlint/pull/1800/commits/0e0ad028dd744b6c0d0f28a7eae1e08b77746711)

But if you think we don't have that much information for a brief explanation
please tell me to fix it, or feel free to fix and push it to this branch 😄

Reply:
The code samples are ok. But I do want to use the label `ktlint` and `disallowed` instead of `with wrapping` and `without wrapping`. So I have changed that.